$MirOS: ports/sysutils/pxegrub/patches/patch-stage2_shared_h,v 1.2 2007/01/20 17:16:09 tg Exp $
--- stage2/shared.h.orig	Sat Jun 19 16:39:47 2004
+++ stage2/shared.h	Sun May 13 14:16:01 2007
@@ -359,6 +359,7 @@ extern char *grub_scratch_mem;
 #define memmove grub_memmove
 #define memcpy grub_memmove	/* we don't need a separate memcpy */
 #define memset grub_memset
+#undef isspace
 #define isspace grub_isspace
 #define printf grub_printf
 #define sprintf grub_sprintf
@@ -368,6 +369,7 @@ extern char *grub_scratch_mem;
 #define strstr grub_strstr
 #define memcmp grub_memcmp
 #define strcmp grub_strcmp
+#undef tolower
 #define tolower grub_tolower
 #define strlen grub_strlen
 #define strcpy grub_strcpy
@@ -848,6 +850,7 @@ typedef enum
   KERNEL_TYPE_BIG_LINUX,	/* Big Linux.  */
   KERNEL_TYPE_FREEBSD,		/* FreeBSD.  */
   KERNEL_TYPE_NETBSD,		/* NetBSD.  */
+  KERNEL_TYPE_OPENBSD, 		/* OpenBSD and MirBSD.  */
   KERNEL_TYPE_CHAINLOADER	/* Chainloader.  */
 }
 kernel_t;
